crypto/x509/x509_test.cc

```diff
@@ -2301,6 +2302,32 @@ TEST(X509Test, Ed25519Sign) {
   ASSERT_TRUE(SignatureRoundTrips(md_ctx.get(), pub.get()));
 }
 
+TEST(X509Test, DilithiumSignVerifyCert) {
```

Comment:
```suggestion
TEST(X509Test, Dilithium3SignVerifyCert) {
```

Reply:
fixed in [df911c5](https://github.com/awslabs/aws-lc/pull/759/commits/df911c5c7aa4e9b77462edae00b95aeab12dc5ef)